app/javascript/components/pages/Jobs.jsx

```diff
@@ -110,7 +108,6 @@ const Job = ({ title, description, imageUrl, company, link, location, createdAt
                         Apply
                     </a>
                 </Button>
-                <div>{`Posted ${postedAtString(createdAt)}`}</div>
```

Comment:
Removing the "posted at" part of the job listing for now to prevent jobs from looking out of date.